decidim-admin/app/views/decidim/admin/authorization_workflows/index.html.erb

```diff
@@ -1,17 +1,17 @@
 <% add_decidim_page_title(t("authorization_workflows", scope: "decidim.admin.titles")) %>
 
 <div class="item_show__header">
-  <h2 class="item_show__header-title">
+  <h1 class="item_show__header-title">
     <%= t("authorization_workflows", scope: "decidim.admin.titles") %>
-  </h2>
+  </h1>
 </div>
 
 <div class="card">
   <%= cell("decidim/verifications/revocations", @authorizations) %>
   <div class="card-divider">
-    <h2 class="card-title">
+    <h1 class="card-title">
```

Comment:
I see there is no accessibility impact on having 2 H1 on the page. I will ignore the rest of occurences.

Reply:
Oh, that's a mistake in my side for sure! We should not have two h1 in the same page. They should be unique. I see that it's not a rule enforced by WAI WCAG checker (I don't see this offence there in this page).

But as far as I know in a11y having a unique h1 for page is something that makes it easier to navigate. I can't find the WCAG recommendation, but It's something that we have in our guide:

>  Every page should have only one <h1> element on them to indicate the main topic of that page